components/speedreader/resources/panel/components/main-panel/index.tsx

```diff
@@ -3,40 +3,93 @@ import * as React from 'react'
 import * as S from './style'
 import Toggle from '$web-components/toggle'
 import { FontStyleList, ContentList } from '../lists'
-import ZoomControl from '../zoom-control'
+import FontSizeControl from '../font-size-control'
 import ThemeControl from '../theme-control'
+import { SiteSettings, Theme, FontSize, ContentStyle, FontFamily } from '../../api/browser'
+import BackgroundSVG from '../../svg/background'
+import { getLocale, splitStringForTag } from '$web-common/locale'
+
+interface MainPanelProps {
+  siteSettings: SiteSettings
+  onThemeChange: (theme: Theme) => void
+  onFontSizeChange: (fontSize: FontSize) => void
+  onContentStyleChange: (contentStyle: ContentStyle) => void
+  onFontFamilyChange: (fontFamily: FontFamily) => void
+  onToggleChange: (isOn: boolean) => void
+}
+
+function MainPanel (props: MainPanelProps) {
+  const braveSpeedreaderOffText = splitStringForTag('braveSpeedreaderTurnOffDesc')
```

Comment:
For some reason this is not displaying correctly
<img width="354" alt="image" src="https://user-images.githubusercontent.com/741836/200503590-9b9aa55c-6120-40ae-bfb5-b46ba6c9573d.png">

Reply:
missed getLocale call there, fixed